### PR TITLE
Init breakthrough

### PR DESCRIPTION
**Pull request by DenEdwards**

### CODE REVIEW

File: src/components/IdeaCard.js [hunk 1]
```diff
@@ -1,66 +1,66 @@
-import React from "react";
+import React from 'react';
```
Comment (vladislab) [inline src/components/IdeaCard.js]: can you revert this file before merging

**Review by vladislab — CHANGES_REQUESTED** [review]

**Review by vladislab — DISMISSED** [review]

**Comment by vladislab** [comment]
@DenEdwards conflicted oof

**Comment by DenEdwards** [comment]
That wasn't showing up two days ago?

**Comment by vladislab** [comment]
idk i approved then it popped up

**Review by vladislab — APPROVED** [review]

**Pull request closed without merging**